src/rpc/blockchain.cpp

```diff
@@ -2470,6 +2470,60 @@ static RPCHelpMan dumptxoutset()
     };
 }
 
+static RPCHelpMan getblocklocations()
+{
+    return RPCHelpMan{"getblocklocations",
+                "\nEXPERIMENTAL warning: this call may be removed or changed in future releases.\n"
+                "\nReturns a JSON for the location of 'blockhash' and its previous 'nblocks' block and undo data.\n",
+                {
+                    {"blockhash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash"},
+                    {"nblocks", RPCArg::Type::NUM, RPCArg::Optional::NO, "Number of block locations to return"},
```

Comment:
I'd like to see this description clarified. Something like: "The number of blocks to process, including the block with given blockhash. The blocks following the given block will be returned if bigger than one."

I'm not entirely satisfied with my wording either, so will be happy for clearer version. The idea is to communicate what exactly is in the result.

Reply:
Updated the documentation:
```
getblocklocations "blockhash" nblocks

EXPERIMENTAL warning: this call may be removed or changed in future releases.

Returns a JSON for the file system location of 'blockhash' block and undo data.

It is possible to return also the locations of previous blocks, by specifying 'nblocks' > 1.

Arguments:
1. blockhash    (string, required) The block hash
2. nblocks      (numeric, required) Maximum number locations to return (up to genesis block)

Result:
[           (json array)
  n,        (numeric) blk*.dat/rev*.dat file index
  n,        (numeric) block data file offset
  n,        (numeric) undo data file offset (if exists)
  "hex",    (string) previous block hash
  ...
]

Examples:
> bitcoin-cli getblocklocation "00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09" 10
```
WDYT?

Reply:
Looks good. :+1: